tests/tests/test-class-alerts.php

```diff
@@ -254,9 +262,45 @@ function test_load_alerts_settings_bad_alert_type() {
 			$this->_handleAjax( 'load_alerts_settings' );
 		} catch ( \WPAjaxDieContinueException $e ) {
 			// We expected this, do nothing.
-			$exception = $e;
 		}
 
+		// TODO: This returns an empty 'success => true' response. It should probably return a failure response instead - 400 Bad Request?
```

Comment:
I think a borked request should return an `HTTP 400 - Bad Request` instead of a 200 with a `success => true` message.